### DISCUSSION TITLE
Sign in with Apple/Other sign-in providers

### DISCUSSION THREAD

**Discussion by jtpotato** [monkeytypegame/monkeytype#2194] [Ideas]
There's firebase docs here: https://firebase.google.com/docs/auth/ios/apple
although it does require someone to join the Apple Developer Program, so I understand if it's outside the scope of the project. Sign in with Discord is more open, although Firebase doesn't have any official docs on it. I did find an example here: https://github.com/luizkc/firebase-discord-oauth2-example

I think it'd be a good option for people that don't want to use Google's service.

**Comment by unrealapex** [comment]
That would be a nice feature. I'm not sure if Miodec has Apple sign in planned, but he was working on GitHub sign in earlier this year.
